deploy/crds/pagerduty_v1alpha1_pagerdutyintegration_cr.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: pagerduty.openshift.io/v1alpha1
```

Comment:
Oh yes, this is just an example. I think the operator-sdk put a dummy one here when generating the initial CRD, and I just updated it as I updated the corresponding CRD. I'll move it to a separate directory to make it less confusing (and in case your question is based on everything in `deploy/crds/` being deployed (this is background knowledge that I don't have)).